test/test_podman_baseline.sh

```diff
@@ -113,7 +113,7 @@ podman images
 ########
 # Create Fedora based container
 ########
-image=$(podman pull fedora)
+image=$(podman pull fedora:28)
```

Comment:
we should be using the latest image, not f28

Reply:
Now using `registry.fedoraproject.org/fedora:29`

Reply:
Now using `registry.fedoraproject.org/fedora:latest` since originally it was `docker.io/library/fedora`.